cloudnet-examples/src/main/java/de/dytanic/cloudnet/examples/sign/ExampleSigns.java

```diff
@@ -16,23 +16,38 @@
 
 package de.dytanic.cloudnet.examples.sign;
 
+import com.google.common.base.Strings;
 import de.dytanic.cloudnet.driver.CloudNetDriver;
-import de.dytanic.cloudnet.ext.signs.AbstractSignManagement;
-import de.dytanic.cloudnet.ext.signs.Sign;
-import de.dytanic.cloudnet.ext.signs.SignLayout;
-import de.dytanic.cloudnet.ext.signs.bukkit.event.BukkitCloudSignInteractEvent;
-import de.dytanic.cloudnet.ext.signs.configuration.entry.SignConfigurationTaskEntry;
+import eu.cloudnetservice.cloudnet.ext.signs.Sign;
+import eu.cloudnetservice.cloudnet.ext.signs.SignManagement;
+import eu.cloudnetservice.cloudnet.ext.signs.bukkit.event.BukkitCloudSignInteractEvent;
+import eu.cloudnetservice.cloudnet.ext.signs.configuration.SignConfigurationEntry;
+import eu.cloudnetservice.cloudnet.ext.signs.configuration.SignConfigurationEntry.KnockbackConfiguration;
+import eu.cloudnetservice.cloudnet.ext.signs.configuration.SignGroupConfiguration;
+import eu.cloudnetservice.cloudnet.ext.signs.configuration.SignLayout;
+import eu.cloudnetservice.cloudnet.ext.signs.configuration.SignLayoutsHolder;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
 import org.bukkit.Material;
 import org.bukkit.event.EventHandler;
+import org.jetbrains.annotations.NotNull;
 
 public final class ExampleSigns {
 
   // getting the SignManagement via CloudNet's service registry
-  private final AbstractSignManagement signManagement = CloudNetDriver.getInstance().getServicesRegistry()
-    .getFirstService(AbstractSignManagement.class);
+  private final SignManagement signManagement = CloudNetDriver.getInstance().getServicesRegistry()
+    .getFirstService(SignManagement.class);
```

Comment:
Is this change possible?

Reply:
Why should it not be possible?

Reply:
See https://github.com/CloudNetService/CloudNet-v3/blob/development/cloudnet-common/src/main/java/de/dytanic/cloudnet/common/registry/DefaultServicesRegistry.java#L47 

The class which get's registered is in the backing map (in this case AbstractSignManagement), but you try to read a SignManagement. This behaviour should be changed in the future by rewriting the lookup in the registry but for now this change will not work.

Reply:
How does this work than: https://github.com/CloudNetService/CloudNet-v3/blob/eb8144865381261b6b5c996b8873cb908678e260/cloudnet-modules/cloudnet-signs/src/main/java/eu/cloudnetservice/cloudnet/ext/signs/bukkit/BukkitSignManagement.java#L50

Reply:
My mistake i was thinking about the old sign management.